index.js

```diff
@@ -186,8 +186,18 @@ exports.viewAccount = async function (options) {
     if (state && state.amount) {
         state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
     }
-    console.log(`Account ${options.accountId}`);
-    console.log(inspectResponse.formatResponse(state));
+    const storageInM=state.storage_usage/1e+6;
+    const accountDetails={
+        'Account:':`${options.accountId} at block ${state.block_height} (https://explorer.testnet.near.org/blocks/${state.block_hash})`,
```

Comment:
I would say that the link to the block is redundant. I can not come up with any usecases.

Reply:
Ok. I will remove that